<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List_Best</class>
 <widget class="QWidget" name="List_Best">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Berlin Sans FB&quot;; color: rgb(175,238,238);
background-color: rgb(204, 204, 204, 0);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
